trie/verkle_iterator.go

```diff
@@ -97,9 +97,9 @@ func (it *verkleNodeIterator) Next(descend bool) bool {
 		it.current = it.stack[len(it.stack)-1].Node
 		it.stack[len(it.stack)-1].Index++
 		return it.Next(descend)
-	case *verkle.HashedNode:
```

Comment:
this code was so old that we were still using the version of `HashedNode` that was storing commitments (and a potential stem as well)